src/script/lua_api/l_server.cpp

```diff
@@ -45,6 +45,17 @@ int ModApiServer::l_get_server_status(lua_State *L)
 	return 1;
 }
 
+// print(text)
+int ModApiServer::l_print(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	std::string text;
+	text = luaL_checkstring(L, 1);
+	Server *server = getServer(L);
+	server->printToConsoleOnly(text);
```

Comment:
Couldn't you compress the contents of this function into: `getServer(L)->printToConsoleOnly(luaL_checkstring(L, 1));` ??